### ISSUE TITLE
Request: Add alt text per file on Mastodon

### ISSUE THREAD

**Issue by tux93** [mvdicarlo/postybirb-plus#142]
Currently it seems that alt text is either only added to the first / primary file when posting multiple files to Mastodon. [1]

Each file should have it's own alt text field to apply different text to each attachment.

[1] This is my assumption based on there only being only one alt text field even when attaching multiple files, I did not actually send a post to find out whether the same text is applied to all or only to the first.  
If there already is a way to apply different alt text per file in postybirb I would be happy to learn how to do it.
